### ISSUE TITLE
Create a BarChart Component

### ISSUE THREAD

**Issue by irshadjsr21** [girlscript-blr/BuildWithGSBlr-Raahat#27]
- This component should use Chart.js 
- This component should create a chart of type: bar
- This component should take the following props
    - labels: list of strings
    - record: list of integer values to display as per labels
    - backgroundColour: single color to be passed as per choice
    - title: The title of the chart
    - colour: The color of the chart
- Create the component as shown in the design.
- The component should be responsive so that it looks good in both desktop and mobile view.
- The `title` props value should be the title of the chart.

Reference Link
https://www.createwithdata.com/react-chartjs-dashboard/

For Design Reference [Click Here](https://www.figma.com/file/TkpKfOPVtQ2XjADFp9Hhbz/Build-with-GS-BLR-Raahat?node-id=372%3A156)

**Location to save the component ```src->components->common->barChart```**

**Comment by kaushik27mishra** [comment]
Can you assign this issue to me?

**Comment by Gautam-Arora24** [comment]
May I work on this issue?

**Comment by irshadjsr21** [comment]
Hola Coder @kaushik27mishra ! 
It's time to start working on the issue assigned to you⏳ 
Please raise the PR till `31st October`⏱ for getting evaluated. And don't forget to register at here: https://tinyurl.com/buildwithgsblr to earn the exciting badges🎖


Feel free to post your queries in the comments thread. 
Also, you can discuss at Google groups: codewithgsblr-forum@googlegroups.com or email us at girlscriptblr@gmail.com

**Comment by kaushik27mishra** [comment]
Hey @irshadjsr21 ,

I have almost completed making the component and it works as expected, but the problem is I have used [react-chartjs-2](https://github.com/jerairrest/react-chartjs-2) package to make this component, however, I checked the [reference link](https://www.createwithdata.com/react-chartjs-dashboard/) later on which you have provided on the issue itself, which doesn't uses that npm package. So I am a little bit confused now! 

Shall I proceed ahead my way or I should implement it in the way mentioned in your reference link.

Regards,
Kaushik
